ballerina/tests/mock_servers.bal

```diff
@@ -102,3 +103,11 @@ service on new Listener(PORT6) {
         });
     }
 }
+
+service on new Listener(PORT7) {
+    remote function onBytes(readonly & byte[] data) returns (readonly & byte[])|Error? {
```

Comment:
Shall we add a assertion here also to see if `data` is `readonly` ?

Reply:
Please check my comment https://github.com/ballerina-platform/module-ballerina-udp/pull/391/files#r798200464